docs/source/debug.rst

```diff
@@ -0,0 +1,53 @@
+Ideas for ‘debugging’ problems in Isca
+======================================
+
+When Isca runs crash it can be difficult to diagnose the problem. We suggest trying these steps below in order to work out where the crash is happening. Not all may be applicable to your problem.
+
+Write Statements
+----------------
+Add in some write commands into the Fortran files you suspect are causing trouble. These can be either to print out values to check if they seem sensible, or periodic statements (e.g. every 100 lines or after every major function) so see how far the code is running. A quick and dirty way of printing variables directly to the terminal is to use a command in the code like this (note this will print every timestep without further conditions):
+
+.. code-block:: fortran
+
+    write(6,*) "<Some description>: ", <variable>
+
+Alternatively variables can be written to files, such as this `example <http://www.python.org/>`_.
+
+Running on 1 core
+-----------------
+A lot of issues on Isca can stem from parallelisation across multiple computing cores. If you run a job on 1 core and it works, you know that the problem lies here. Quite often it can be the allocation of the start/end integers ``is, ie, js, je``
+
+Column Model
```

Comment:
I'd suggest removing the reference to the column model for now as it isn't in the master yet